gdbgui/static/js/gdbgui.js

```diff
@@ -2601,27 +2602,33 @@ const Expressions = {
     handle_changelist: function(changelist_array){
         for(let changelist of changelist_array){
             let expressions = state.get('expressions')
+            let expr_type = state.get("expr_type")
             , obj = Expressions.get_obj_from_gdb_var_name(expressions, changelist.name)
 
             if(obj){
-                if('new_children' in changelist){
-                    let expr_type = state.get("expr_type")
-                    let new_children = changelist.new_children.map(child_obj => Expressions.prepare_gdb_obj_for_storage(child_obj, expr_type))
-                    obj.children = obj.children.concat(new_children)
-                }
-                if('value' in changelist && obj.expr_type === 'expr'){
-                    // this object is an expression and it had a value updated.
-                    // save the value to an array for plotting
-                    if(changelist.value.indexOf('0x') === 0){
-                        obj.can_plot = true
-                        obj.values.push(parseInt(changelist.value, 16))
-                    }else if (!window.isNaN(parseFloat(changelist.value))){
-                        obj.can_plot = true
-                        obj.values.push(changelist.value)
+                if(changelist['has_more'] == 1 && 'name' in changelist){
+                    // already retrieved children of obj
```

Comment:
add more info to comment: `already retrieved children of obj, but more fields were added. Re-fetch the object from gdb`